helm/aws-load-balancer-controller/templates/service.yaml

```diff
@@ -1,17 +1,42 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "aws-load-balancer-controller.namePrefix" . }}-webhook-service
+  name: {{ template "aws-load-balancer-controller.fullname" . }}
   namespace: {{ .Release.Namespace }}
   {{- with .Values.serviceAnnotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   labels:
-{{ include "aws-load-balancer-controller.labels" . | indent 4 }}
+    {{- include "aws-load-balancer-controller.labels" . | nindent 4 }}
+spec:
+  ports:
+    - port: 443
```

Comment:
cert won't be valid for this service, so opening this port does not serve any purpose.

Reply:
Good point, I'll remove this for now.